docs/docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,215 @@
+---
+id: quickstart
+title: Quickstart
+description: A cheat-sheet of Neabri commands for returning users.
+---
+
+This quickstart is a Nebari commands reference for experienced and returning users.
+If you're new to Nebari, start at [Installing Nebari](installing-nebari.md).
+
```

Comment:
We can add the Nebari CLI diagram here, when it's ready.

Reply:
WIP here: https://lucid.app/lucidchart/ebfd0c6f-34e4-4817-8d28-79b448fe7cd8/edit?viewport_loc=35%2C147%2C1579%2C916%2C0_0&invitationId=inv_e2e6a631-c4cc-4b6c-9a26-4f900790f4fb

Reply:
WIP here: https://lucid.app/lucidchart/ebfd0c6f-34e4-4817-8d28-79b448fe7cd8/edit?viewport_loc=75%2C149%2C1579%2C916%2C0_0&invitationId=inv_e2e6a631-c4cc-4b6c-9a26-4f900790f4fb